### PR TITLE
 mautrix-telegram: 0.5.2 -> 0.6.0

### PR DESCRIPTION
**Pull request by nyanloutre**
###### Motivation for this change
Draft pull request in prevision of mautrix-telegram 0.6.0

Also update:
 * mautrix-appservice: 0.3.10.dev1 -> 0.3.11
 * telethon: 1.6.2 -> 1.9.0 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/servers/mautrix-telegram/default.nix [hunk 1]
```diff
@@ -1,14 +1,16 @@
-{ lib, python3, mautrix-telegram }:
+{ lib, python3, fetchFromGitHub, mautrix-telegram }:
 
 with python3.pkgs;
 
 buildPythonPackage rec {
   pname = "mautrix-telegram";
-  version = "0.5.2";
+  version = "0.6.0-rc1";
```
Comment (matthiasbeyer) [inline pkgs/servers/mautrix-telegram/default.nix]: I'm not sure whether we want -rc releases in nixpkgs, do we?

Comment (nyanloutre) [reply pkgs/servers/mautrix-telegram/default.nix]: It's only a draft PR to test the new version, I will update it when stable is out

**Comment by nyanloutre** [comment]
Stable is out, this can be merged now

**Comment by Ma27** [comment]
Nice! I'll deploy that later that evening to my Matrix HS and merge this if everything goes well :)

**Comment by Ma27** [comment]
Deployed to my Matrix HS, upgrade went well, I can still send/receive messages :+1:

**Review by Ma27 — APPROVED** [review]

**Comment by Ma27** [comment]
@nyanloutre thanks!

**Comment by Ma27** [comment]
Oh btw @nyanloutre do you think that we should backport this?

**Comment by nyanloutre** [comment]
Yes you could but the module #63589 will not be available in 19.03

**Comment by Ma27** [comment]
> Yes you could but the module #63589 will not be available in 19.03

Yeah, I know that :)
But there are probably some more people like me who are running `mautrix-telegram` with a custom service.

**Comment by Ma27** [comment]
Tested and backported as ee942c3e2c64096dc7e7df6f1a65ab79c70fd27e, a2e438685a4201beb17696eb298c29e14e280e32, 41d303c3425212c23860e4cb9e8c4a07378c7f52, 28e64db237dbe1e0d430cb61e7372223b49635b9

**Pull request merged**